### ISSUE TITLE
ProgressIndicator: Indeterminate ProgressIndicator should have proper accessibility

### ISSUE THREAD

**Issue by LiangMingChen** [OfficeDev/office-ui-fabric-react#8805]
## Codepen demonstrating the issue
Indeterminate ProgressIndicator

## Actual behavior:

https://github.com/OfficeDev/office-ui-fabric-react/blob/bf912f0679f7c54bfd6864c6e8b7a9dfc2de128b/packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx#L89

https://developer.microsoft.com/en-us/fabric#/components/progressindicator

![image](https://user-images.githubusercontent.com/3789716/56529412-8c06ab80-6505-11e9-9449-af23e47ceb96.png)

## Expected behavior:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role

Specifically

If the progressbar is describing the loading progress of a particular region of a page, the author SHOULD use aria-describedby to point to the status, and set the aria-busy attribute to true on the region until it is finished loading. It is not possible for the user to alter the value of a progressbar because it is always readonly.

**Comment by apneer** [comment]
Thank you for filing this issue. ProgressIndicator component is indeed missing 'describedBy' and 'busy' aria properties. This is a bug. We'll provide solution for this shortly.

**Comment by Anschur** [comment]
Not sure I understood this bug - sorry if this is not the right place to ask questions... Is the region supposed to be describedby the progressbar, or vice versa? From the text above it seems at least that the aria-busy should be set on the region, and not the progressbar. And it might make sense that the region is described by the progress as well?

**Comment by apneer** [comment]
Hi Anschur. Thank you for taking a look. Looking closer to the spec, it seems that the props would belong to the enclosing region rather than component. In this case, this would not be a case of missing properties on ProgressIndicator but usage issue.

@LiangMingChen, can you see try setting aria-busy and aria-describeby on the region that contains the progressbar? Would this solve the issue?

**Comment by LiangMingChen** [comment]
Yes, let me open a work item on our team to work through it.   Thanks for looking into this.
